### PR TITLE
add more templates/make pages lil more responsive/getStatusFromHeaders

### PR DESCRIPTION
**Pull request by FirstDayAtWork**

### CODE REVIEW

File: static/scripts/app-index.js
Comment (Lifeissimple-zxc) [inline static/scripts/app-index.js:1]: What are these for?

File: static/scripts/template-script.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const burgerButton = document.querySelector('.burger-btn');
+const navBarContent = document.querySelector('.navbar-content');
+const navBarLinks = document.querySelectorAll('.navbar-links');
+const content = document.querySelector('.container');
+const dropdownMenu = document?.querySelector('.dropdown');
+const dropDownContent = document?.querySelector('.dropdown-content');
+
+// show/hide navbar content
+burgerButton?.addEventListener('click', () => {
```
Comment (Lifeissimple-zxc) [inline static/scripts/template-script.js:9]: burger?

File: static/scripts/template-script.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        dropDownContent.style.display = 'none'
+    } else if(dropDownContent.style.display === 'none' 
+                || dropDownContent.style.display === ''){
+        dropDownContent.style.display = 'block'
+    }
+
+})
+
+
+// dropdownMenu?.addEventListener('mouseover', () => {
```
Comment (Lifeissimple-zxc) [inline static/scripts/template-script.js:37]: Do we need this in production code?

File: templates/base.html [hunk 3]
```diff
@@ -15,19 +15,26 @@
...
+            <div class="burger-container">
+                <button class="burger-btn">☰</button>
+            </div>
+            <div class="navbar-content">
             <div class="navbar-left-side">
                 <div class="navbar-links">
-                    <a class="navbar-link-style" href="../pages/index.html">Home</a>
-                    <a class="navbar-link-style" href="../pages/about.html">About</a>
-                    <a class="navbar-link-style" href="../pages/donate.html">Donate</a>
+                    <a class="navbar-link-style" href="/">Home</a>
```
Comment (Lifeissimple-zxc) [inline templates/base.html:27]: Should it be /index instead?

File: templates/donate.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<!-- donate.html -->
+{{ define "content" }}
+    <h1>Dono Page</h1>
```
Comment (Lifeissimple-zxc) [inline templates/donate.html:3]: Spelling is wrong

File: views/const.go [hunk 5]
```diff
@@ -3,16 +3,24 @@ package views
 const (
 	templatesFolder     string = "templates"
 	TemplateCSS         string = "../static/styles/template.css"
-	LoginCSS            string = "../static/styles/login-style.css"
+	LoginCSS            string = "../static/styles/style.css"
```
Comment (Lifeissimple-zxc) [inline views/const.go:6]: Good decision with the name!

File: static/scripts/app-index.js
Comment (FirstDayAtWork) [reply static/scripts/app-index.js:1]: for stuff that will happen in the future!

File: static/scripts/app-register.js [hunk 6]
```diff
@@ -73,13 +73,10 @@ formbtn.addEventListener('click', async (e) => {
...
 
-            
-
-            
-
-        if(result.status_code === 200) {
-                console.log('OK')
-        } else if(result.status_code === 409){
+        if(datafetch.ok) {
+            console.log(datafetch.status)
```
Comment (Lifeissimple-zxc) [inline static/scripts/app-register.js:77]: Why not make it a switch statement on response status?

File: static/scripts/template-script.js (hunk 1 above)
Comment (FirstDayAtWork) [reply static/scripts/template-script.js:9]: hamburger* :)

File: static/styles/template.css [hunk 7]
```diff
@@ -95,3 +201,135 @@ header{
...
+    display: none;
+} */
+
+.noscript-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
```
Comment (Lifeissimple-zxc) [inline static/styles/template.css:216]: I did not know about this, interesting technique!

File: templates/base.html (hunk 3 above)
Comment (FirstDayAtWork) [reply templates/base.html:27]: yep, i just copy/paste it from my second project, anyway this links not working properly
i can't fix it
but u can!

File: static/scripts/template-script.js (hunk 2 above)
Comment (FirstDayAtWork) [reply static/scripts/template-script.js:37]: kinda...i can del em

File: templates/base.html (hunk 3 above)
Comment (Lifeissimple-zxc) [reply templates/base.html:27]: yeah ok, closing

File: static/scripts/template-script.js (hunk 2 above)
Comment (Lifeissimple-zxc) [reply static/scripts/template-script.js:37]: Let's agree not to have commented code when pushing to PROD.

File: static/scripts/template-script.js (hunk 1 above)
Comment (Lifeissimple-zxc) [reply static/scripts/template-script.js:9]: So we keep it with this name?

Comment (FirstDayAtWork) [reply static/scripts/template-script.js:9]: yep why not ? it's right name for that thing
google burger/hambuger menu css

File: views/const.go (hunk 5 above)
Comment (FirstDayAtWork) [reply views/const.go:6]: rename it coz this file applies to every page
prob shoud rename LoginCSS to something like template ?

File: static/scripts/app-register.js (hunk 6 above)
Comment (FirstDayAtWork) [reply static/scripts/app-register.js:77]: coz i use switch for short tasks like:
switch(something){
case 1:
return test (1-2 lines)
case 2:
return test2
...

```
but in our example u return :
let user_err = document.createElement('small');
                    user_err.classList.add('user-err');
                    user_err.innerText = `Username ${userNameInput.value} is already exist`;
                    userNameInput.style.border = '2px solid #b90909'
                    // remove err if already exist
                    if(userNameInput.nextElementSibling.classList.contains('user-err')){
                        userNameInput.nextElementSibling.remove();
                    }

                    userNameInput.after(user_err);
                    setTimeout(() => {
                        user_err.remove();
                        userNameInput.style.border = '2px solid #dddddd';
                        userNameInput.addEventListener('focus', () => userNameInput.style.border = '2px solid #0941b9');
                        userNameInput.addEventListener('blur', () => userNameInput.style.border = '2px solid #dddddd')
                    }, 5000);
            console.log(datafetch.statusText, 'Username already exist!')
```

            which is not fit!

File: static/styles/template.css (hunk 7 above)
Comment (FirstDayAtWork) [reply static/styles/template.css:216]: u def should try this game https://flexboxfroggy.com/ for better understanding FLEX!

Comment (Lifeissimple-zxc) [reply static/styles/template.css:216]: Saved this link, thanks

**Pull request merged**